src/server/server_family.cc

```diff
@@ -1270,6 +1281,15 @@ void PrintPrometheusMetrics(const Metrics& m, DflyCmd* dfly_cmd, StringResponse*
   AppendMetricWithoutLabels("lua_blocked_total", "", m.lua_stats.blocked_cnt, MetricType::COUNTER,
                             &resp->body());
 
+  AppendMetricWithoutLabels("backup_count", "", m.loading_stats.backup_count, MetricType::COUNTER,
```

Comment:
naming convetion. ever increasing counters in open metrics must end with `_total`